client/js/spotify-gdpr.js

```diff
@@ -1,5 +1,40 @@
 // @ts-check
-/* global c3 */
+import {getStreamingData, collateStreamingData} from './analysis/gdpr.js';
+
+const {c3, zip} = window;
+
+zip.workerScriptsPath = '/js/zip/';
+
+const btnUpload =
+  /** @type {HTMLButtonElement} */
+  (document.getElementById('btn-data-upload'));
+
+btnUpload.addEventListener('click', async () => {
+  const inputUpload =
+      /** @type {HTMLInputElement} */
+      (document.getElementById('input-data-upload'));
+
+  inputUpload.click();
+
+  await new Promise(
+      (resolve) => inputUpload.addEventListener(
+          'change',
+          resolve,
+          {once: true},
+      ),
+  );
+
+  if (!inputUpload.files) return;
+
+  const fileUpload = inputUpload.files.item(0);
+  if (!fileUpload) return;
+
+  const records = await getStreamingData(fileUpload);
+  console.log(records);
```

Comment:
remove debug statements

Reply:
if i don't have these then the variable isn't used for anything (which will trip off eslint)
also this pull request doesn't even represent finished code, `client-style-5` will remove the console logs
is that okay?

Reply:
It's common to use `delete` in these cases:
```
// Records are currently unused.
delete records;
```

Reply:
ESLint says: `Parsing error: Deleting local variable in strict mode`, and my IDE highlights everything in red when I try to do that...

Reply:
Instead, I just put an ESLint ignore comment